packages/vitest/src/integrations/css/css-modules.ts

```diff
@@ -0,0 +1,19 @@
+import type { CSSModuleScopeStrategy } from '../../types'
+
+export function generateCssFilenameHash(filename: string) {
+  return Buffer.from(filename).toString('base64').substring(0, 6)
```

Comment:
This is probably a bad hash function as first characters will be the same for different files in the same folder.